app/components/sections/users/user-table/component.js

```diff
@@ -0,0 +1,48 @@
+import Ember from "ember";
+import computed from "ember-computed-decorators";
+import UserValidations from "last-strawberry/validators/user";
+import Roles from "last-strawberry/constants/roles";
+
+export default Ember.Component.extend({
+  session: Ember.inject.service(),
+
+  classNames: "col stretch",
+
+  roles: _.values(Roles),
```

Comment:
I have tried it once. It does not works due to the options value of ember power select require an array